src/styles/index.scss

```
@@ -1,3 +1,128 @@
 body {
   margin: 0;
 }
+
+.card {
+  width: 198px;
+  padding: 32px 0 16px;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  color: #060B35;
+  font-size: 14px;
+  border: 1px solid #F3F3F3;
+  border-radius: 5px;
+
+
+  &__top {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  &__imac {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__img {
+    display: flex;
+    max-width: 100%;
+  }
+}
+
+.product {
+  padding-inline: 16px;
+
+  &__name {
+    padding-top: 40px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__code {
+    color: #616070;
+    font-size: 10px;
+    padding-top: 4px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-top: 16px;
+  }
+
+  &__rating--stars {
+    display: flex;
+
```

make use of sass nesting for bem
```suggestion
  &__rating {
    display: flex;
    justify-content: space-between;
    align-items: center;
    padding-top: 16px;
  &--stars {
    display: flex;
}
}
```
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**